src/app/pages/analyzers/analyzers-management/analyzers-table/analyzers-table.component.html

```diff
@@ -0,0 +1,6 @@
+<nb-card>
+  <nb-card-body>
+    <ng2-smart-table [settings]="tableSettings" [source]="tableSource">
+    </ng2-smart-table>
+  </nb-card-body>
+</nb-card>
```

Comment:
This won't work since this template is not defined in the component. Moreover, this is a part of what's already there.

Reply:
removed in #119